dist/htmlminifier.js

```diff
@@ -1503,7 +1514,22 @@
                 });
               }
             }
+            if (prevTag && inlineTextTags(prevTag.charAt(0) === '/' ? prevTag.substr(1) : prevTag)) {
+              text = collapseWhitespace(text, options, /(?:^| )$/.test(currentChars));
+            }
             text = prevTag || nextTag ? collapseWhitespaceSmart(text, prevTag, nextTag, options) : trimWhitespace(text);
+            if (!text && / $/.test(currentChars) && prevTag && prevTag.charAt(0) === '/') {
```

Comment:
I'm thinking of collapsed whitespaces always getting turned into ` ` - but I guess no harm making them `\s`